devicetypes/smartthings/zwave-basic-heat-alarm.src/zwave-basic-heat-alarm.groovy

```diff
@@ -0,0 +1,165 @@
+/**
+ *  Copyright 2018 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+	definition(name: "Z-Wave Basic Heat Alarm", namespace: "smartthings", author: "SmartThings") {
+		capability "Temperature Alarm"
+		capability "Sensor"
+		capability "Battery"
+		capability "Health Check"
+
+		//zw:S type:0701 mfr:026F prod:0001 model:0002 ver:1.07 zwv:4.24 lib:03 cc:5E,86,72,5A,73,80,71,85,59,84 role:06 ff:8C01 ui:8C01
+		fingerprint mfr: "026F ", prod: "0001", model: "0002", deviceJoinName: "FireAngel Thermistek ZHT-630 Heat Alarm/Detector"
+	}
+
+	simulator {
+		status "battery 100%": "command: 8003, payload: 64"
+		status "battery 5%": "command: 8003, payload: 05"
+		status "HeatNotification": "command: 7105, payload: 00 00 00 FF 04 02 80 4E"
+		status "HeatClearNotification": "command: 7105, payload: 00 00 00 FF 04 00 80 05"
+		status "HeatTestNotification": "command: 7105, payload: 00 00 00 FF 04 07 80 05"
+	}
+
+	tiles(scale: 2) {
+		multiAttributeTile(name: "heat", type: "lighting", width: 6, height: 4) {
+			tileAttribute("device.heat", key: "PRIMARY_CONTROL") {
+				attributeState("cleared", label: "cleared", icon: "st.alarm.smoke.clear", backgroundColor: "#ffffff")
+				attributeState("detected", label: "HEAT", icon: "st.alarm.smoke.smoke", backgroundColor: "#e86d13")
+				attributeState("tested", label: "TEST", icon: "st.alarm.smoke.test", backgroundColor: "#e86d13")
+			}
+		}
+		valueTile("battery", "device.battery", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+			state "battery", label: '${currentValue}% battery', unit: ""
+		}
+
+		main "heat"
+		details(["heat", "battery"])
+	}
+}
+
+def installed() {
+	// Device checks in every hour, this interval allows us to miss one check-in notification before marking offline
+	sendEvent(name: "checkInterval", value: 2 * 60 * 60 + 2 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+
+	def cmds = []
+	createHeatEvents("heatClear", cmds)
```

Comment:
If this only ever returns one event, you can make these lines more concise.

Reply:
Thanks. I will correct that.